app/components/customer-visible-icon/component.js

```diff
@@ -1,9 +1,5 @@
-import Component from "@ember/component";
-import { attributeBindings } from "@ember-decorators/component";
-import classic from "ember-classic-decorator";
+import Component from "@glimmer/component";
 
-@classic
-@attributeBindings("title")
 export default class CustomerVisibleIcon extends Component {
   title = "This project's comments are visible to the customer";
```

Comment:
Can't we make this a template only component by moving the title directly into the template?